praetorian_cli/sdk/chariot.py

```
@@ -170,31 +180,31 @@ def patch(self, type: str, body: dict = None, params: dict = None) -> dict:
         process_failure(resp)
         return resp.json()
 
-    def delete_by_key(self, type: str, key: str, body: dict = {}, params: dict = {}) -> dict:
-        self.delete(type, body | dict(key=key), params)
+    def delete_by_key(self, type: str, key: str, body: dict | None = None, params: dict | None = None) -> dict:
```

Ruff (0.15.15)

[error] Function argument `type` is shadowing a Python builtin

(A002)